packages/apps/dev-wallet/src/pages/select-profile/select-profile.tsx

```diff
@@ -11,15 +14,46 @@ export function SelectProfile() {
     <main>
       <Box margin="md">
         <Heading variant="h5">Select a profile</Heading>
-        {profileList.map((profile) => (
-          <div key={profile.uuid}>
-            <Link to={`/unlock-profile/${profile.uuid}`}>{profile.name}</Link>
-          </div>
-        ))}
-        <br />
-        <Link to="/create-profile">
-          <Text bold>Create profile</Text>
-        </Link>
+        <Stack
+          flexDirection="row"
+          alignItems="center"
+          padding={'sm'}
+          gap={'sm'}
+          flexWrap="wrap"
+        >
+          {profileList.map((profile) => {
+            return (
+              <div key={profile.uuid}>
+                <Link
+                  to={`/unlock-profile/${profile.uuid}`}
+                  style={{ textDecoration: 'none' }}
+                >
+                  <div
+                    className={cardClass}
+                    style={{ backgroundColor: idToColor(profile.uuid) }}
```

Comment:
I would say instead of having this call here, we can do it in profile creation. and then store it in database as profile.accentColor. then we use it here. later we let users to pick another color themselves.